CONTRIBUTING.md

```diff
@@ -5,28 +5,111 @@ This project is very new, so we'll probably revise these guidelines. Please
 comment on a bug before putting significant effort in, if you'd like to
 contribute.
 
-Guidelines:
+## Guidelines
 
 * Follow the Style Guide (see below)
 * Keep work branches in your own github fork, rebase your own branches at will
 * Squash or rebase branches before merging to master so the commits make sense
-  on their own
+on their own
 * Get a SGTM from someone relevant before merging
 * Keep commits to master bisect safe (i.e. each commit should pass all tests)
 * Sign-off commits before merging (see below)
 * Make sure your commit message references the issue or bug number, if there is
-  one, identifies the reviewers, and follows a readable style, with the long
-  description including any additional information that's likely to help future
-  spelunkers.
-  For example:
-  ```
-  Frobnicate the URL bazzer before flattening pilchard. Fixes #6. r=mossop,rnewman
-
-  The frobnication method used is as described in Podder's Miscellany, page 15.
-  Note that this pull request doesn't include tests, because we're bad people.
-
-  Signed-off-by: Random J Developer <random@developer.example.org>
-  ```
+one, identifies the reviewers, and follows a readable style, with the long
+description including any additional information that's likely to help future
+spelunkers (see below)
+
+```
+Frobnicate the URL bazzer before flattening pilchard, r=mossop,rnewman. Fixes #6.
+
+The frobnication method used is as described in Podder's Miscellany, page 15.
+Note that this pull request doesn't include tests, because we're bad people.
+
+Signed-off-by: Random J Developer <random@developer.example.org>
+```
+
+## Example
+
+* Fork this repo.
+
+* Clone your fork locally:
```

Comment:
Unless I'm missing something, the first bullet point is "Fork this repo." Good idea about adding the link.